### PR TITLE
Small Dockerfile fix

### PR DESCRIPTION
**Pull request by andresilva91**
## Motivation

Small fix

## Proposal

Missing line break

## Test Plan

Run it

### CODE REVIEW

**Comment by andresilva91** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1132** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1132" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/linera-io/linera-protocol/1132?utm_source=stack-comment).

**Review by christos-h — APPROVED** [review]
Good catch

File: docker/Dockerfile.aarch64 [hunk 1]
```diff
@@ -58,7 +58,7 @@ RUN cargo build --release \
     --features scylladb \
     && strip target/aarch64-unknown-linux-gnu/release/linera \
     && strip target/aarch64-unknown-linux-gnu/release/linera-proxy \
-    && strip target/aarch64-unknown-linux-gnu/release/linera-server
+    && strip target/aarch64-unknown-linux-gnu/release/linera-server \
```
Comment (ma2bd) [inline docker/Dockerfile.aarch64:61]: @christos-h @andresilva91 IIRC one wouldn't need `\` if `&&` was at the end of the line.

Comment (andresilva91) [reply docker/Dockerfile.aarch64:61]: Yeah, you're right. There was this bash formatter that I was using that preferred it this way for some reason, so I just left it like this 😅
We could change it though, if you prefer the other way

Comment (ma2bd) [reply docker/Dockerfile.aarch64:61]: That was just FYI. Keep using your favorite formatter! :)

**Pull request merged**